src/EditorFeatures/Core/Extensibility/Commands/ExportInteractiveCommandAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.CodeAnalysis.Editor
+{
+    [MetadataAttribute]
+    [AttributeUsage(AttributeTargets.Class)]
+    internal class ExportInteractiveAttribute : ExportAttribute
```

Comment:
Do we need a specialized type here?  I imagine there must be some similar implementation somewhere else in the Roslyn codebase.
(if so, it's fine to clean this up in a subsequent checkin)